Todo.py

```diff
@@ -0,0 +1,203 @@
+
+            if reaction.channel_id == int(channelId): #Check if it's a todo-message (check if it's the good channel)
+                if len(message.embeds) > 0: # Check if it's an embed, I think this will avoid most problems
+                    if reaction.emoji.name == EMOJIS['wastebasket']:
+                        await bot.get_channel(reaction.channel_id).delete_messages([message])
+
+                        repostFieldValue = None
+                        for field in message.embeds[0].fields:
+                            if field.name == "Public repost":
+                                repostFieldValue = field.value
+                        
+                        if repostFieldValue != None:
+                            repostMessage = await bot.get_channel(int(repostFieldValue.split(':')[0][2:-2])).fetch_message(int(repostFieldValue.split(':')[1][1:]))
+                            await repostMessage.delete()
+                    elif reaction.emoji.name == EMOJIS['check']:
+                        await message.remove_reaction(EMOJIS['hourglass'], bot.user)
+                    elif reaction.emoji.name == EMOJIS['hourglass']:
+                        await message.remove_reaction(EMOJIS['check'], bot.user)
+
+    @commands.Cog.listener()
+    async def on_raw_reaction_remove(self, reaction):
+        message = await bot.get_channel(reaction.channel_id).fetch_message(reaction.message_id)
+
+        with open(TODO_CHANNEL, "r") as file:
+            channelId = file.readlines()[0]
+
+        if reaction.channel_id == int(channelId): # Check if it's a todo-message (check if it's the good channel)
+            if len(message.embeds) > 0: # Check if it's an embed, I think this will avoid most problems
+                if reaction.user_id != bot.user.id:
+                    if reaction.emoji.name == EMOJIS['check']:
+                        await message.add_reaction(EMOJIS['hourglass'])
+                    elif reaction.emoji.name == EMOJIS['hourglass']:
+                        await message.add_reaction(EMOJIS['check'])
+
+    @commands.group()
+    async def todo(self, ctx):
+        '''Commands to manage a todolist.'''
+        if ctx.invoked_subcommand is None:
+            await ctx.send('Error: See for ``' + PREFIX + 'help todo``')
+
+    @todo.command()
+    async def add(self, ctx, *args):
+        '''Command to add a todo. Usage : <the thing todo>;<type of todo>;<assigned to / false>;<repost public channel / false>'''
+        with open(TODO_CHANNEL, "r") as file:
+            channelId = file.readlines()[0]
+            if channelId == None or channelId == "":
+                await ctx.channel.send("The todos' channel must be selected with the command " + PREFIX + "todo channel")
+                return
+        channel = bot.get_channel(int(channelId))
+
+        command = ""
+        for arg in args:
+            command += " {}".format(arg)
+        command = command.split(";")
```

Comment:
Maybe letting users issue commands with multiple args only using spaces would make it more natural to them. This would make it closer to how the CLI works as well as most other bots on discord. It also wouldn't cause any additional difficulty and would probably be clearer.

Reply:
Hum, the problem is that I need to keep spaces for todo's descriptions. Otherwise they have to use quotes.

Reply:
That is true. Although a way to fix this would be to put the message at the end. Like this all words after the commands would be part of the message. What do you think ?